test/mod.test.js

```diff
@@ -0,0 +1,17 @@
+import test from 'ava'
+import {mod} from '../src'
+
+test('a dividend larger than divisor', t => {
+  const dividend = 12
+  const divisor = 8
+  const expected = 4
+  const result = mod(dividend, divisor)
+  t.is(expected, result)
+})
+
+test('a negative dividend', t => {
+  const dividend = -12
+  const divisor = 8
+  const expected = 4
+  const result = mod(dividend, divisor)
```

Comment:
Oh man that was sloppy. Sorry about that. I thought I had swapped out that documentation.  I have made those updates. Thanks!